src/client/transport/local.js

```diff
@@ -121,6 +121,8 @@ export class Local {
    */
   subscribe() {}
 
+  subscribeGameMetadata() {}
```

Comment:
Change line 89 in this file to also reflect the new `gameMetadata` argument that's sent back (even if it doesn't really do anything in the local mode).